Automattic/wp-calypso
# Hosting: Move to one single SFTP user per site model

#### Changes proposed in this Pull Request

Updates model to one SFTP user per site by refactoring all selectors to use the endpoint changes developed in D35883. 

It also leaves the data-layer ready for whenever we move to a model supporting multiple SFTP users managed by all admins.

#### Testing instructions

* Apply D35883-code and sandbox the API.
* Go to a new site that is eligible for atomic hosting.
* Check that the Activate SFTP button appears in the sftp card.
* Click the button and check that a username (site domain) and password are returned
* Reload the page and check that previously created username is displayed
* Check that the reset password button works
* Log in to the same site with a different admin user and check that the previously created sftp username is visible.

Fixes #37891
